src/workerd/api/pyodide/pyodide-test.c++

```diff
@@ -29,14 +29,14 @@ KJ_TEST("getPythonSnapshotRelease") {
   featureFlags.setPythonWorkersDevPyodide(true);
   {
     auto res = KJ_ASSERT_NONNULL(getPythonSnapshotRelease(featureFlags));
-    KJ_ASSERT(res.getPyodide() == "0.26.0a2");
+    KJ_ASSERT(res.getPyodide() == "dev");
     KJ_ASSERT(res.getFlagName() == "pythonWorkersDevPyodide");
   }
 
   featureFlags.setPythonWorkers(false);
   {
     auto res = KJ_ASSERT_NONNULL(getPythonSnapshotRelease(featureFlags));
-    KJ_ASSERT(res.getPyodide() == "0.26.0a2");
+    KJ_ASSERT(res.getPyodide() == "dev");
     KJ_ASSERT(res.getFlagName() == "pythonWorkersDevPyodide");
   }
 
```

Comment:
The next one should also be "dev"
You're missing the change to pyodide.c++
https://github.com/cloudflare/workerd/pull/3988/files#diff-7a09d0821698c77d11ae25dcb62c95991f65cc7df549015499ce154cc2e3705eR555-R573